src/Skybrud.Umbraco.BorgerDk/BorgerDkService.cs

```diff
@@ -72,23 +86,27 @@ public partial class BorgerDkService {
                     // Make the call to the database
                     var dto = scope.Database.FirstOrDefault<BorgerDkArticleDto>(sql);
 
+                    borgerDkCacheRefresher.Refresh(dto);
+
                     // Return the article
                     return dto?.Meta;
 
                 } catch (Exception ex) {
-                    _logger.LogError(ex, "Unable to insert redirect into the database");
-                    throw new Exception("Unable to insert redirect into the database", ex);
+                    _logger.LogError(ex, "Unable to get article from the database");
+                    throw new Exception("Unable to get article from the database", ex);
                 }
-
             }
-
         }
 
-        public void Import(BorgerDkArticle article) {
+        public void Import(BorgerDkArticle article, bool useCache = false) {
```

Comment:
Calling this method should always update the article, so the `useCache` parameter should be removed.